package.json

```diff
@@ -15,8 +15,8 @@
     "gasLimit": "5400000"
   },
   "scripts": {
-    "test": "cross-conf-env run-with-ganache --ganache-cmd ganache-cli --gasLimit npm_package_config_gasLimit 'npm run truffle compile && npm run truffle migrate && npm run truffle test'",
-    "ganache": "cross-conf-env ganache-cli -l npm_package_config_gasLimit --account=\"0x0191ecbd1b150b8a3c27c27010ba51b45521689611e669109e034fd66ae69621,9999999999999999999999999999999999999999999\" --account=\"0x00f360233e89c65970a41d4a85990ec6669526b2230e867c352130151453180d,9999999999999999999999999999999999999999999\" --account=\"0x987a26abca7432016104ce2f24ce639340e25afe06ac69f68791399e7a5d1028,9999999999999999999999999999999999999999999\" --account=\"0x89af34b1b7347834048b99423dad174a14bf14540d720d72c16ae92e94b987cb,9999999999999999999999999999999999999999999\" --account=\"0xc867be647eb2bc51e4c0d61066859875cf3634fe949b6f5f85c69ab90e485b37,9999999999999999999999999999999999999999999\" --account=\"0xefabcc2377dee5e51b5a9e65a3854aec85fbbec3cb584d8ad4f9679869fb33c6,9999999999999999999999999999999999999999999\"",
+    "test": "cross-var run-with-ganache --ganache-cmd ganache-cli --gasLimit $npm_package_config_gasLimit 'npm run truffle compile && npm run truffle migrate && npm run truffle test'",
```

Comment:
whats wrong with cross-conf-env ?

Reply:
Doesn't work on Windows.

Reply:
windows is not supported by arc.

Reply:
Is there any reason not to make this simple change that will make my life easier (not to mention anyone else who would like to run the tests under windows)?

Reply:
It will make the project much complex to be verified and tested on different envs .

Reply:
I'm am not suggesting, by making this code change, that we change our policy on supporting Windows.  At the same time, I don't think our policy should be to actively resist improving the Windows experience if we are handed a trivial opportunity to do so.